packages/apps/tools/src/components/Common/DrawerToolbar/index.tsx

```diff
@@ -65,10 +64,11 @@ export const DrawerToolbar: ForwardRefExoticComponent<
         <>
           <div className={expandedDrawerTitleClass}>
             {sections[visibleSection].title}
-            <IconButton
+            <Button
```

Comment:
This looks different then it used to look now

Reply:
@isaozler is the right person here

Reply:
I will resolve these now and we can fix them separately later